crates/sdk/src/prover/agg.rs

```diff
@@ -264,7 +264,7 @@ where
 
     #[cfg(feature = "evm-prove")]
     #[instrument(name = "agg_layer", skip_all, fields(group = "root", idx = 0))]
-    fn generate_root_proof_impl(
+    pub fn generate_root_proof_impl(
```

Comment:
do we really need to make this pub? I hoped to avoid it

Reply:
changed it back to private